src/StatsPlots.jl

```diff
@@ -20,7 +20,7 @@ import Clustering: Hclust, nnodes
 using Interpolations
 import MultivariateStats: MDS, eigvals, projection, principalvars,
                             principalratio, transform
-
+import AbstractFFTs: fft, ifft
```

Comment:
Importing implies we overload them, which we don't.
```suggestion
using AbstractFFTs: fft, ifft
```

Reply:
I agree, though importing anyway is the convention here.